internal/jobservice/repository/sql_job_service_test.go

```diff
@@ -282,8 +301,6 @@ func TestHealthCheck(t *testing.T) {
 // This test will fail if sqlite writes are not serialised somehow due to
```

Comment:
Is comment still correct?  I guess now we serialized and enable WAL?

Reply:
The locking in conjunction with WAL seems to eliminate them, at least in my testing.